appspec.yml

```diff
@@ -4,12 +4,15 @@ files:
  - source: /
    destination: /home/datamade/la-metro-dashboard-deployment-root
 hooks:
- BeforeInstall:
+  ApplicationStop:
```

Comment:
Excellent catch here. Looks like the Flask template in `deploy-a-site` doesn't include this step by default and I forgot to add it:  https://github.com/datamade/deploy-a-site/blob/master/templates/flask/%7B%7Bcookiecutter.directory_name%7D%7D/appspec.yml As a quick note, the PR body says this PR adds `ApplicationStart` instead of `ApplicationStart`, might be helpful for posterity to adjust that.

Reply:
Yeah, I noticed that about the templates – it's because the previous process is killed after starting the new one in `app_start` in the zero downtime paradigm. ☠️

Updated the description – great catch!